src/components/Vault/VaultStrategy.tsx

```diff
@@ -125,16 +125,16 @@ const VaultStrategyModal: FC<VaultProps & ModalBaseProps> = ({
                   <Percentage>{fees.data?.withdrawFee ?? "0"}</Percentage>
                 </Skeleton>
               </dd>
-              <dt>Management</dt>
+              <dt>Performance</dt>
               <dd className="text-right">
                 <Skeleton isLoading={fees.isLoading}>
-                  <Percentage>{fees.data?.platformFee ?? "0"}</Percentage>
+                  <Percentage>{fees.data?.performanceFee ?? "0"}</Percentage>
```

Comment:
The problem here was the value displayed in the performance row was the management value and the management value was the performance value. Is there a parsing issue somewhere??